Content.Shared/Strip/Components/ThievingComponent.cs

```diff
@@ -19,4 +19,11 @@ public sealed partial class ThievingComponent : Component
     [ViewVariables(VVAccess.ReadWrite)]
     [DataField("stealthy")]
     public bool Stealthy;
+
+    /// <summary>
+    ///  Should the user be able to see hidden items? (i.e pockets)
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite)]
+    [DataField("ignoreStripHidden")] //DeltaV
```

Comment:
again just [DataField] its automatically rw and uses camelCase variable name

= false can be removed thats the default

also nothing uses it so...

Reply:
Can you phrase this? I don't know what you want me to do

Reply:
delete those lines and have

```cs
[DataField]
public bool IgnoreStripHidden; // DeltaV
```

then make it actually used by aghost and felenid prototype, but felenids should still show doafter